app/javascript/packs/validators/validators_map.vue

```diff
@@ -134,23 +112,29 @@
     },
     methods: {
       position_horizontal: function(longitude) {
-        return 50 + (longitude / 160 * 50) + '%';
+        if(longitude < 0) {
```

Comment:
perhaps `let division_factor = longitude < 0 ? 142 : 146` and than single return would look cleaner